phases/account.go

```diff
@@ -1,9 +1,106 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"net/http"
 
-// Account ...
-func Account() (string, error) {
-	fmt.Println("SetAccount")
-	return "", nil
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type organizationData struct{
+	Name string
+	Slug string
+}
+
+func isValid(choice int, limit int ) bool {
+	return choice >= 1 && choice <= limit
+}
+
+// Account returns the slug of the selected account. If the user selects
+// the personal account, the slug is empty.
+func Account(apiToken string) (string, error) {
+	req, err := http.NewRequest(http.MethodGet, "https://api.bitrise.io/v0.1/organizations", nil)
+	if err != nil {
+		return "", err
+	}
+
+	req.Header.Set("Authorization", "token " + apiToken)
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return "", err
+	}
+
+	if res.StatusCode != 200 {
+		return "", fmt.Errorf("fetch orgs: %s", res.Status)
+	}
+
+	m := struct{
```

Comment:
no need to initialize, simply `var m struct{....}`

Reply:
It is here, as the response json is in format {"data": {...} }